tools/format/format.conf

```diff
@@ -895,6 +895,7 @@ emit:chance
 :countdown_interval
 :countdown_destroy
 :countdown_action
+:drop_action
```

Comment:
* Need to provide for specifying actors, updating lines similar to:
`:(use_action(:@)?|countdown_action|drop_action)`

* Need to specify `emit_actor` properties themselves